partials/settings-page.php

```diff
@@ -30,19 +30,19 @@
 		function ( $a, $b ) {
 			if ( $a['order'] < $b['order'] ) {
 				return -1;
-			} elseif ( $a['order'] == $b['order'] ) {
+			}
+			if ( $a['order'] === $b['order'] ) {
 				return 0;
-			} else {
-				return 1;
 			}
+			return 1;
```

Comment:
The `else` statement here was simply not necessary because the previous line returns early if the condition is true.